[refman] Mention Ltac2 in intro. (#11324)
coq/coq: doc/sphinx/introduction.rst

   of the formalism. Chapter :ref:`themodulesystem` describes the module
   system.

> -  The second part describes the proof engine. It is divided in seven

Reviewer: ```suggestion
-  The second part describes the proof engine. It is divided into several
```

Better to be nonspecific about the number of chapters--otherwise it may become inaccurate.

   chapters. Chapter :ref:`vernacularcommands` presents all commands (we
   call them *vernacular commands*) that are not directly related to
   interactive proving: requests to the environment, complete or partial
   evaluation, loading and compiling files. How to start and stop
   proofs, do multiple proofs in parallel is explained in
   Chapter :ref:`proofhandling`. In Chapter :ref:`tactics`, all commands that
   realize one or more steps of the proof are presented: we call them

>    *tactics*. The legacy language to combine these tactics into complex proof

Reviewer: ```suggestion
   *tactics*. The original language to combine these tactics into complex proof
```

I think it's premature to call ltac "legacy" when ltac2 is still considered experimental.

Author: The point is to already start discouraging people from using it, at least those who are not afraid by the experimental status of Ltac2. Despite being experimental, Ltac2 should still provide a better and more robust experience to people wanting to develop complex proof strategies.

>    strategies is given in Chapter :ref:`ltac`. The new and experimental

Reviewer: ```suggestion
   strategies is given in Chapter :ref:`ltac`. The currently experimental
```

IMHO "new" doesn't add any information given that it is experimental and "eventually going to replace Ltac".

>    language that is eventually going to replace Ltac is presented in

Reviewer: ```suggestion
   language that will eventually replace Ltac is presented in
```
